### DISCUSSION TITLE
User not authorized

### DISCUSSION THREAD

**Discussion by ApolloTheSearcher** [community/community#13448] [Copilot Conversations] [answered]
Suddenly when I want to change my github account in vscode when I restart vscode, then this message appears against my github copilot, whereas before I've never had a problem like this, I've been using this github copilot for almost 2 years without any problems and suddenly there is a problem like this "User not Authorized", I've tried to log out and then log in , and restart vscode , I've been trying for almost a whole day to find this solution, then got no,
I use MacOs
![Jepretan Layar 2022-03-25 pukul 00 44 07](https://user-images.githubusercontent.com/99478967/159978027-dfb7ef8c-5761-45d8-bd67-330e69763474.png)

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by ApolloTheSearcher** [reply]
not working dave

**Reply by davecheney** [reply]
Can you please try going to https://github.com/settings/applications, and revoking the authorisation for
![Screen Shot 2022-03-25 at 1 01 36 pm](https://user-images.githubusercontent.com/7171/160040005-71608ca4-e03f-43b2-9f98-87bc2c052169.png)
Then
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by ApolloTheSearcher** [reply]
![Jepretan Layar 2022-03-25 pukul 09 20 57](https://user-images.githubusercontent.com/99478967/160041833-c0758497-d75f-4af2-8863-9bcb4be2a6d1.png)
no github copilot

**Reply by ApolloTheSearcher** [reply]
sign in for sync or only sign githubcopilot?

**Comment by davecheney** [comment]
Hmm, according to our database [ApolloTheSearcher](https://github.com/ApolloTheSearcher) joined the waitlist last month. I'm not sure how you have been using copilot before then, do you have an alternative github account?

**Reply by davecheney** [reply]
Yes, I've confirmed @[ApolloTheSearcher](https://github.com/ApolloTheSearcher) is still on the waitlist.

**Reply by ApolloTheSearcher** [reply]
yes dave, before i have one account github can running the githubcoplito, i delete account that and change to this account, I don't think there will be any problems like this

**Reply by davecheney** [reply]
OK, this should be fixed. Check ur email.

**Reply by ApolloTheSearcher** [reply]
Thanksssssss Daveee lovveeee Yiuuuuuuuu

**Comment by ApolloTheSearcher** [comment, marked as answer]
I've tried many ways but none of them worked, I think I should wait for the github copilot waiting list for this new account, I GIVE UP😭😭

**Reply by davecheney** [reply]
[ApolloTheSearcher](https://github.com/ApolloTheSearcher) is enabled to use copilot. Log out / Log in, usual thing.

**Reply by ApolloTheSearcher** [reply]
![Jepretan Layar 2022-03-25 pukul 09 45 50](https://user-images.githubusercontent.com/99478967/160044640-7c9d52e2-5d3e-4f1a-9531-aa364d80d8d1.png)
OMG 😱  Thanks Dave Love Youuuuu, Since I used this I really helped.
